keycloak-oauth-policy/src/main/java/io/apiman/plugins/keycloak_oauth_policy/beans/KeycloakOauthConfigBean.java

```diff
@@ -101,7 +101,7 @@
      * Delegate Kerberos Ticket
      */
     @JsonProperty("delegateKerberosTicket")
-    private boolean delegateKerberosTicket;
+    private boolean delegateKerberosTicket = false;
```

Comment:
Note: false is the default value for a boolean field, so this isn't really necessary.